### PR TITLE
Card Sort Form #2

### PR DESCRIPTION
**Pull request by samanehsan**
Ticket: https://openscience.atlassian.net/browse/EOSF-52
## Changes
- Add card sort form where users can sort the answers from the first card-sort into nine categories
- For all forms (overview, free response, card-sort, card-sort-2) make continue button send data to route & show the next form
- Show free response answers about the situation at the top of the two card sort forms  
- Change card count color based on number of cards in bucket -- black for too few cards, green for correct amount of cards and red for too many cards  
  screen shot 2016-06-21 at 11 27 45 am

### CODE REVIEW

File: app/components/card-sort-2/template.hbs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                  <div class="item alert alert-info" data-id={{card.id}}>{{card.content}}</div>
+                {{/draggable-object}}
+              {{/each}}
+            </div>
+          {{/draggable-object-target}}
+      </div>
+    {{/each-in}}
+</div>
+<div class="row">
+    {{#each-in buckets2 as |group categories|}}
```
Comment (samchrisinger) [inline app/components/card-sort-2/template.hbs]: 👍 looks much better!

**Comment by samchrisinger** [comment]
Overall, looks great. I'm wondering what it would take to merge the card sort components into a single component... the code is really similar with some small exceptions. Does this seem possible?

**Comment by samanehsan** [comment]
@samchrisinger, the component templates are really similar but I don't see how they could be combined into one. However, I can see how the card sort 1 buckets could be broken out into a separate component since it is used in both card sort templates.

**Comment by samchrisinger** [comment]
Do we have a use case for rendering either of the card sorts independently of the other (initially the second sort could just be hidden)? Maybe these could just be unified into a single component?

**Comment by samanehsan** [comment]
I can look into combining them - it might be possible with a lot of conditionals to check what should be rendered for that form.

**Pull request merged**